src/solutions/LiquidityIncentive/components/SelectPointType.tsx

```diff
@@ -0,0 +1,66 @@
+import {
+  Collapse,
+  Divider,
+  FormControl,
+  FormErrorMessage,
+  HStack,
+  Input,
+  Stack,
+  Text,
+} from "@chakra-ui/react"
+import ExistingPointsTypeSelect from "components/[guild]/RolePlatforms/components/AddRoleRewardModal/components/AddPointsPanel/components/ExistingPointsTypeSelect"
+import useGuild from "components/[guild]/hooks/useGuild"
+import RewardImagePicker from "platforms/SecretText/SecretTextDataForm/components/RewardImagePicker"
+import { useFormContext, useWatch } from "react-hook-form"
+import Star from "static/icons/star.svg"
+import { PlatformType } from "types"
+
+const SelectPointType = () => {
+  const { guildPlatforms } = useGuild()
+  const {
+    control,
+    register,
+    formState: { errors },
+  } = useFormContext()
```

Comment:
We could make the whole component more type safe by providing a generic type to this hook ;)